banzai/lco.py

```diff
@@ -485,6 +488,18 @@ def _init_saturate(image):
                          image=image)
             raise MissingSaturate
 
+    @staticmethod
+    def _update_spectral_sections(hdu):
+        """
+        Manually update data and trim sections for fs01 spectral camera
+        :param hdu: Astropy ImageHDU
+        """
+        section_keywords = {'TRIMSEC': '[2:2046,3:2015]',
```

Comment:
You might add a case to only set these values when the trimsec and datasec are their current values.